buildkite-agent.rb

```diff
@@ -154,7 +154,10 @@ def plist
         <true/>
 
         <key>KeepAlive</key>
-        <true/>
+        <dict>
+          <key>SuccessfulExit</key>
+          <true/>
```

Comment:
I think you want `<false/>`?

Stopping an agent via the Web UI causes the agent to exit successfully, with a 0 exit status. With SuccessfulExit set to `true` this will cause launchd to restart the agent, to keep it alive on successful exit. I think instead we want launchd to keep the agent alive _until_ a successful exist.

Reply:
🤦‍♂️